src/types/User.ts

```diff
@@ -0,0 +1,5 @@
+export interface User {
```

Comment:
Now the name `User` is well understood in the type file itself, but when we reuse this type, it will be more convenient to give a name that will directly inform that it is a type, interface, enam, and so on. (For example: `IUser`, `UserInterface`, `TUser`, `UserType`).